sources/datasources/stories.js

```diff
@@ -0,0 +1,80 @@
+/** @format */
+
+const Events = require('../models/event.js');
+const Stories = require('../models/story.js');
+const currentStories = require("../models/currentStory");
+const Users = require('../models/user.js');
+const Clubs = require('../models/club.js');
+const Venues = require('../models/venue.js');
+const AccessLevel = require('../models/accessLevel.js');
+const { DataSource } = require('apollo-datasource');
+const { ApolloError } = require('apollo-server');
+
+class StoryAPI extends DataSource {
+	constructor() {
+		super();
+	}
+    initialize(config) {}
+    getStories(args) {
+		return Stories.find(args);
+	}
+	async getCurrentStories() {
+        let activeStories=await currentStories.find();
+        return activeStories;        
+	}
+	async addStory(story) {
+		let retPromise = {};
```

Comment:
retPromise?
meaning?

Reply:
We need to return a promise to the resolver when this function is called

Reply:
the variable name I am asking, 'retPromise'

Reply:
short for returnPromise?

Reply:
Yes it is short form for return Promise

Reply:
put better name please. we will graduate soon and we want people to understand code. retPromise is weird